docs/rules.md

```diff
@@ -13,9 +13,14 @@ When running under Wayland with X11 libraries loaded some features will not be a
 When running under Wayland without X11 libraries loaded even more features will not be available.
 Rule features known not to work under Wayland include process and mouse process conditions.
 Under Wayland using keyboard groups may result in incorrect symbols being input for simulated input.
-Under Wayland simulating inputs when modifier keys are pressed may result in incorrect symbols being input.
+Under Wayland simulating inputs when modifier keys are pressed may result in incorrect symbols being sent.
 Simulated input uses Xtest if available under X11 or uinput if the user has write access to /dev/uinput.
-To get access to /dev/uinput run `sudo setfacl -m u:${user}:rw /dev/uinput`*
+The easiest way to maintain write access to /dev/uinput is to use Solaar's alternative udev rule by downloading
+`https://raw.githubusercontent.com/pwr-Solaar/Solaar/master/rules.d-uinput/42-logitech-unify-permissions.rules`
+and copying it as root into the `/etc/udev/rules.d` directory.
+You may have to reboot your system for the write permission to be set up.
+Another way to get write access to /dev/uinput is to run `sudo setfacl -m u:${user}:rw /dev/uinput`
```

Comment:
I think that `${USER}` is the default way of accessing current username i most common shells (uppercase).

Reply:
Indeed.  In tcsh either works.

Reply:
In bash only the uppercased version works.